.github/workflows/release.yml

```diff
@@ -0,0 +1,83 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        description: "Tag for the next release. Ex.: v5.0.0"
+
+env:
+  NEXT_RELEASE_TAG: ${{ github.event.inputs.tag }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+permissions:
+  contents: write
+
+jobs:
+  release-script-test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./tools/release
+
+    steps:
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+      - name: Cache Python venv
+        uses: actions/cache@v3
+        with:
+          path: ./tools/release/venv
+          key: venv-${{ hashFiles('tools/release/requirements.txt') }}
+      - name: Install dependencies
+        run: |
+          python3 -m venv venv
+          . ./venv/bin/activate
+          pip install -r requirements.txt
+      - name: Run tests
+        run: |
+          . ./venv/bin/activate
```

Comment:
Why do we have this `.` in the beginning?

Reply:
> source and . are synonymous in Bash.

see https://askubuntu.com/a/25491